### DISCUSSION TITLE
[FIXED] [FEATURE REQUEST] Bring back the axis fix from stepmania

### DISCUSSION THREAD

**Discussion by kfrew** [TeamRizu/OutFox#419] [03 - Feature Requests]
For my pad + generic Ps2 2 port adapter usb adapter (and presumably others) the only way without complex mapping/drivers (see here):

https://www.stepmania.com/forums/input-adapters-and-controllers/show/483

There was a feature in previous stepmanias:
Options > Input options > Advanced Input Selections > Axis Fix 

Might make sense to add this option when Legacy input is enabled

**Comment by Scraticus** [comment]
We will have a look and see if it still works. Does it do so on 5.1/5.0?

**Reply by kfrew** [reply]
Yes it does ! Was the only way to get my adapter to work -- I tried a lot of drivers.

**Comment by Scraticus** [comment]
Fixed in Alpha 4.16-> Alpha V
